frontend/src/features/frame/components/sidebar/SideBar.tsx

```diff
@@ -21,6 +21,7 @@ import {
   Compare as CompareIcon,
   WatchLater as WatchLaterIcon,
   ListAlt as ListIcon,
+  Stars as StarsIcon,
```

Comment:
Nice choice! Did you have other candidate icons you hesitated with? I find that `StarRate` or `Grading` may be good too

Reply:
Another possibility is `LocalActivity`.  
`Grading` may look too similar to "My comparisons", don't you think?